core-meetings/2019-11-oss-meeting.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Maintenance: November 2018 Core Meeting
```

Comment:
I dont think we should call this "core". Can we call this "contributors" meeting ? We don't want to make this exclusive

Reply:
We always called them Core meeting (check the other notes) - and they are exclusive, since participation is limited to the developers in the Slack 🤷‍♂️

Reply:
And the paragraph right below this title clearly explains the nature of the core, so I don't see any issue with this wording.

Reply:
That said, what could be done is:
1. come up with a clearer set of rules that defines what is a "core" contributor (good reference: https://www.youtube.com/watch?v=376dz1oIr3g)
1. reword across the whole repo this concept, reflecting what decided in point 1

Reply:
I've also created a dedicated discussion for this -> https://github.com/react-native-community/discussions-and-proposals/issues/59